openml/datasets/functions.py

```diff
@@ -862,7 +866,7 @@ def edit_dataset(
 
       Returns
       -------
-      data_id of the existing edited version or the new version created and published"""
+      data_id of the edited dataset """
```

Comment:
"data_id" -> Dataset id

The documentation does not need to expose variable names internal to the function.